Src/BootCamp.Chapter/Equipment.cs

```diff
@@ -7,58 +8,82 @@ namespace BootCamp.Chapter
     /// </summary>
     public class Equipment
     {
-        private Weapon _weapon;
+        private float _totalWeight;
+        private float _totalDefense;
+        private float _totalAttack;
+
+        public Weapon Weapon { get; set; }
```

Comment:
You have duplicate implementation of every setter. Instead of using auto properties, move setter method logic to setter property.